src/App.tsx

```diff
@@ -1,11 +1,9 @@
 import * as React from 'react';
 
-import IHelloProps from 'interfaces/IAppProps';
+import Post from 'post/post';
 
-export default class App extends React.Component<IHelloProps>{
-    public render(): JSX.Element {
-        return (
-            <h1>{this.props.greet} To {this.props.company}</h1>
-        );
-    }
-}
+export const App: React.FunctionComponent<object> = (): JSX.Element => (
```

Comment:
This should be export default because we are exporting only one module from this file.

Reply:
Okay, I Will do Thanks.